flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableTestBase.java

```diff
@@ -147,6 +153,46 @@ public void createTestTopic(String topic, int numPartitions, int replicationFact
         }
     }
 
+    public <K, V> void produceToKafka(
```

Comment:
Should we produce data to Kafka topic similar to the existing tests such as `KafkaTableITCase#testPerPartitionWatermarkWithIdleSource` for consistency?